macros.in

```diff
@@ -807,7 +807,8 @@ package or when debugging this package.\
   RPM_OPT_FLAGS=\"%{optflags}\"\
   RPM_ARCH=\"%{_arch}\"\
   RPM_OS=\"%{_os}\"\
-  export RPM_SOURCE_DIR RPM_BUILD_DIR RPM_OPT_FLAGS RPM_ARCH RPM_OS\
+  RPM_BUILD_NCPUS=\"%{_smp_build_ncpus}\"\
```

Comment:
Hmm, this should perhaps be %{?_smp_build_ncpus} since that macro comes from the platform-macros and in theory there could be platforms where it's not defined. OTOH might be saner to just move the whole ncpus business to the main macros file, dunno.

Other than that pondering, looks good to me.